tools/ConsoleTypes.natvis

```diff
@@ -105,4 +105,11 @@
             <Item Name="[ptr]">_ptr</Item>
         </Expand>
     </Type>
+
+    <Type Name="til::generational&lt;*&gt;">
+        <DisplayString>{{ generation={_generation._value}}}</DisplayString>

```

Comment:
note spacing
```suggestion
        <DisplayString>{{ generation={_generation._value} }}</DisplayString>
```

is it possible to include the short version of the body in here too?

can we use a short indicator instead of a long one? `Gen.{_generation._value}`